src/System.Net.NetworkInformation/System.Net.NetworkInformation.sln

```diff
@@ -33,10 +33,10 @@ Global
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
 		{3CA89D6C-F8D1-4813-9775-F8D249686E31}.Debug|Any CPU.ActiveCfg = Windows_Debug|Any CPU
 		{3CA89D6C-F8D1-4813-9775-F8D249686E31}.Debug|Any CPU.Build.0 = Windows_Debug|Any CPU
-		{3CA89D6C-F8D1-4813-9775-F8D249686E31}.FreeBSD_Debug|Any CPU.ActiveCfg = FreeBSD_Debug|Any CPU
-		{3CA89D6C-F8D1-4813-9775-F8D249686E31}.FreeBSD_Debug|Any CPU.Build.0 = FreeBSD_Debug|Any CPU
-		{3CA89D6C-F8D1-4813-9775-F8D249686E31}.FreeBSD_Release|Any CPU.ActiveCfg = FreeBSD_Release|Any CPU
-		{3CA89D6C-F8D1-4813-9775-F8D249686E31}.FreeBSD_Release|Any CPU.Build.0 = FreeBSD_Release|Any CPU
+		{3CA89D6C-F8D1-4813-9775-F8D249686E31}.FreeBSD_Debug|Any CPU.ActiveCfg = Windows_Release|Any CPU
+		{3CA89D6C-F8D1-4813-9775-F8D249686E31}.FreeBSD_Debug|Any CPU.Build.0 = Windows_Release|Any CPU
+		{3CA89D6C-F8D1-4813-9775-F8D249686E31}.FreeBSD_Release|Any CPU.ActiveCfg = Windows_Release|Any CPU
+		{3CA89D6C-F8D1-4813-9775-F8D249686E31}.FreeBSD_Release|Any CPU.Build.0 = Windows_Release|Any CPU
```

Comment:
Ah yeah this seems a bit weird. I think I deleted the FreeBSD portions from my solution and then it got merged badly with master.